plugins/ca-python-standard/bin/ca-python-install

```diff
@@ -40,3 +40,19 @@ fi
 poetry -v install || poetry -v install || exit 1
 echo "Devbox setup has completed poetry install"
 pre-commit install --install-hooks || echo "WARNING: could not install pre-commit hooks" >&2
+
+# hook env vars up to cert files for agreeable netskope x python packages
+if [ -d "$HOME" ]; then
+  curl -o $HOME/Downloads/netskope_bundle.pem https://raw.githubusercontent.com/cultureamp/devbox-extras/main/certs/nscacert_combined.pem
+  export CERT="$HOME/Downloads/netskope_bundle.pem"
+  export SSL_CERT_FILE=$CERT
+  # # this one will fix most of the python cli
+  export REQUESTS_CA_BUNDLE=$CERT
+  # # this one is for python sdk for vertex
+  # #!! but it may still throw a missing selected ALPN property error - for python version > 3.9
+  export GRPC_DEFAULT_SSL_ROOTS_FILE_PATH=$CERT
```

Comment:
**issue** I don't think we `source` this script, so these exports probably won't work? but should work in `env` block in `plugin.json`